src/ansys/simai/core/data/training_data.py

```diff
@@ -84,28 +84,33 @@ def get_subset(self, project: Identifiable["Project"]) -> Optional[SubsetEnum]:
                 the :class:`~.projects.Project` object for, or its ID.
 
         Returns:
-            SubsetEnum of the subset that the training data belongs to in the given project.
-                (e.g. <SubsetEnum.VALIDATION: 'Validation'>)
+            None if subset is unassigned or SubsetEnum of the subset that the training data belongs to in the given project.
```

Comment:
```suggestion
            If the subset is unassigned, None; otherwise, the SubsetEnum of the subset to which the training data belongs in the given project.
```

Reply:
```suggestion
            The ``SubsetEnum`` of the subset to which the training belongs to if any, ``None`` otherwise.
```
If you're less lazy than me, you can make SubsetEnum link to the class